### ISSUE TITLE
Can't add/update libraries

### ISSUE THREAD

**Issue by the-lazy-fox** [jellyfin/jellyfin-kodi#511]
**Describe the bug**
I have added 2 libraries successfully but now when I want to update them or add a new one, it always ran into an error.

**To Reproduce**
1. Reset Kodi Local Database from Jellyfin addon
2. Reset jellyfin settings when suggested
3. Kodi restarts
4. Connect to Jellyfin Server
5. Select 1+ Libaries
6. Sync is done successfully
7. Open Jellyfin Addon
8. Try to add another Library or to update the ones just added

**Expected behavior**
It should work as expected

**Logs**
https://pastebin.com/SX4VSmfp

**System (please complete the following information):**
 - OS: LibreElec + Docker
 - Jellyfin Version: 10.7.2
 - Kodi Version: 18.9
 - Addon Version: 0.7.2+py2
 - Playback Mode: Default

**Comment by mcarlton00** [comment]
This should have been fixed by #494.  0.7.2 was just published 24 hours ago, have you tried to sync a new library since updating?

**Comment by the-lazy-fox** [comment]
Hello @mcarlton00 
All of this has been tested and done with 0.7.2+py2 from the official kodi repository.
It seems that the fix is not working as expected :S

**Comment by mcarlton00** [comment]
I can't replicate this.  The fix is working as intended.  I've just tested on 18.9 and 19.0.  I don't know what's going on with yours, but this seems to be an unrelated problem.  You're getting error codes back from the server whenever you try to do a full sync.  Additionally, you somehow are duplicating the libraries that you're trying to sync in the menu.

```
2021-04-28 21:02:19.017 T:2532299632  NOTICE: JELLYFIN.full_sync -> INFO::jellyfin_kodi/full_sync.py:186 starting sync with [u'bb1cb51306ca3bab9a7f98be8fa364d0,5cfba0105d3d1b710c3b12baa847e59d', u'bb1cb51306ca3bab9a7f98be8fa364d0,5cfba0105d3d1b710c3b12baa847e59d', u'Boxsets:Refresh', u'f137a2dd21bbc1b99aa5c0f6bf02a805,767bffe4f11c93ef34b805451a696a4e']
```

`bb1cb51306ca3bab9a7f98be8fa364d0` and `5cfba0105d3d1b710c3b12baa847e59d` are duplicated, and formatted incorrectly.  It should be one ID per entry, but for some reason you have 'bb1cb51306ca3bab9a7f98be8fa364d0,5cfba0105d3d1b710c3b12baa847e59d' as one entry.  And the logs are complaining about that (look at line 44, for example).  Something is different about these libraries.  What kind are they?

**Comment by the-lazy-fox** [comment]
Thanks for helping on that.
I have no idea. When the issue showed up today, I have reset the DB and the plugin completely, started over and I'm back here again.
What can I do to help?

**Comment by mcarlton00** [comment]
I found the problem.  If you add a library from the addon menu root by right clicking/context menuing and selecting 'sync this library', it will work.  But if you go through the 'manage libraries' and select more than one at a time it will fail.  Should have a PR for it in the next few days.

**Comment by Norin-Radd** [comment]
i have the same issue

first was not syncing series, and gave error by the 2 ways, "menu root by right clicking/context menuing ... through the 'manage libraries' ", so I tried to do a repair, it removed the library, but didn't scan, I try to add it, does not sync... i even tried to uninstall the addon and installing the addon maintaining the definitions, same error... i wanted to go back to previous version, but I couldn´t even install it by the repo... 

https://paste.kodi.tv/xuxodenesu

**Comment by mcarlton00** [comment]
This should also be fixed by #513, but there's something weird going on with yours.  This log line makes no sense, since it means that it's effectively trying to sync the same library as both a mixed and a normal library, but over and over and over again.  Is it a mixed library that you're trying to sync?

```
INFO::jellyfin_kodi/full_sync.py:186 starting sync with ['Mixed:7843bb6380e8e6e858637b97d0049d36,d565273fd114d77bdf349a2896867069', 'Mixed:d565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069,Mixed:d565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', '8a1eb8a0816b8a605bd5cf2c70a6255e', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069', 'd565273fd114d77bdf349a2896867069']
```

**Comment by Norin-Radd** [comment]
> ibrary that you're trying to sync?
> 
> ```
> INFO::jellyfin_kodi/full_sync.py:186 starting sync with
> ```

i have 1 folder/library Ao Vivo (live) which is mixed content (movies and series) but just as 1 movie of a comedian show.

**Comment by mcarlton00** [comment]
Found it.  The issue is that we immediately try to add a synced library to the config file, but it doesn't get cleaned up afterwards if it fails.  In most cases this is fine because it asks if you want to continue the sync on next boot, but in the event that the thing was formatted improperly (like it is in 0.7.2) it just gets tacked on over and over again when you try to sync from the menu.  I've added a cleanup step on boot to make sure everything we retrieve from the config file is formatted properly.  This should be fixed in 0.7.3

**Comment by the-lazy-fox** [comment]
Great investigation and fix @mcarlton00 !
Thrilled to test it.

**Comment by Norin-Radd** [comment]
> Found it. The issue is that we immediately try to add a synced library to the config file, but it doesn't get cleaned up afterwards if it fails. In most cases this is fine because it asks if you want to continue the sync on next boot, but in the event that the thing was formatted improperly (like it is in 0.7.2) it just gets tacked on over and over again when you try to sync from the menu. I've added a cleanup step on boot to make sure everything we retrieve from the config file is formatted properly. This should be fixed in 0.7.3

when will happen the release of 0.7.3?

**Comment by mcarlton00** [comment]
We should have it out this week sometime.  I have a few other issues that I'm also trying to find fixes for, and it needs to wait until somebody else on the team reviews my PRs
